### DISCUSSION TITLE
Interruptible Toggles

### DISCUSSION THREAD

**Discussion by waddlesquatt** [AntiMicroX/antimicrox#570] [Q&A]
Is possible to set up a toggle interrupt action like you can on Steam controller options? Example below for those who dont understand

using steam controller config in COD4 / insurgency / stalker anomaly or any fps with ability to lean left or right i have it to where when i Aim down the sights my left and right Dpads inputs switch to Q and E respectively and i press either once to lean and it stay held down until when i release my L1/LT button it reverts back and releases the toggled buttons so i wont be still leaning left or right when i zoom out. 

i have my setup with left and right D-pads as reference:

while Aiming down sight (L1/LT held down [modifier button in the steam config while in the D-pad input options]) and activating my my left Dpad is Q[Toggle][interruptable] to lean left and right Dpad is E [Toggle][interruptable] to lean right

In-Game i do not have the option to toggle Leaning enabled cause it feels weird and requires pressing it again to stop leaning which has gotten me killed alot

ive been trying to figure this out for a good while any help will be appreciated

**Comment by winterandchaiyun** [comment]
已收到，谢谢（这是自动回复）

**Reply by deepindra** [reply]
I also am searching for this, I am using a ebook reader **where I have toggled right click to begin selecting text**.Then the application requires me to hit Enter+ Esc to register to highlight. However I have to toggle off the previous key, which is three keys pressed altogether, I have not good hands, wrists, I wish I could deactivate the toggle in the 2nd shortcut set key rather than toggling it off.
